package.yml

```diff
@@ -14,6 +14,7 @@
   tasks:
     - name: make sure packages are installed
       apt:
+        force_apt_get: yes
```

Comment:
Is this necessary?

Also I couldn't get the script working without adding `update_cache: yes`

Reply:
I built the .deb package on Mac OS X which doesn't have aptitude installed (as far as I know no port for it exists on Mac OS X). I did try to build the .deb package on multiple versions of Ubuntu (16.04, 18.04) but had problems with Vagrant talking to VirtualBox. It was easier to force using apt-get. Whats the "standard" OS for building these .deb packages? Ubuntu version <whatever>? I can revisit getting this working on Ubuntu if you think I should. Also I didn't have to add "update_cache: yes" but it won't hurt to use this option...

Reply:
@kowh-ai Sorry maybe I miss something but the point of using VirtualBox and Vagrant is that the packages are build in a reproducible environment (the target of the distribution) so the host OS should not matter. But anyway, if `force_apt_get` works better let's keep (along `update_cache: yes`)

Reply:
@amercader Agreed that building the package should be done on the target distribution however Iv'e tried may things and haven't got it to work. I was talking to Sergey about this also a couple of days ago and he had the same problem building packages on Ubuntu 18.04. Mac OS X doesn't have an aptitude port AFAIK, hence needed to force "apt-get"